test/tokenlock.js

```diff
@@ -329,7 +329,7 @@ contract("TokenLock", (accounts) => {
     assert.equal(await tokenLock.availableAmount(), 0);
     //each period is 250 seconds
     await utils.increaseTime(350);
-    assert.equal((await tokenLock.sinceStartTime()).toString(), "252");
+    assert.equal((await tokenLock.sinceStartTime()).toString().match("252|253").length, 1);
```

Comment:
Better calculate based on the current time I think

Reply:
wdym?